.evergreen/config.yml

```diff
@@ -1075,6 +1128,14 @@ buildvariants:
            - name: "test-2.4-sharded_cluster"
            - name: "test-2.4-standalone"
 
+- matrix_name: "serverless"
+  matrix_spec:
+    os-requires-32:
+      - "ubuntu-16.04"
```

Comment:
Should this be Ubuntu 18.04? Latest server builds are no longer published on Ubuntu 16.04 (see CDRIVER-3946). It won't matter for serverless. And drivers don't need to copy this. But if drivers are testing as many server variants as possible on as few platforms as they can, 18.04 could be preferable.

Reply:
Updated. I originally used 16.04 since there was no definition in the config for 18.04, but I just went ahead and added it.